src/api/cf.nim

```diff
@@ -26,12 +27,20 @@ const
   ## base url of the forgesvc endpoint
   modsBaseUrl = "https://addons-ecs.forgesvc.net/api/v2"
 
+
+converter getDependencies(json: JsonNode): int =
+  var id: int = json{"id"}.getInt()
+  if id == 0:
+    id = json{"addonId"}.getInt()
```

Comment:
Replace `json{"addonId"}` with `json["addonId"]` because the latter throws if the property can't be found.